recipe/meta.yaml

```diff
@@ -1,42 +1,48 @@
 {% set label = "'full'" %}
 {% set tests = "['pywt']" %}
-{% set name = "PyWavelets" %}
-{% set version = "1.4.1" %}
+{% set name = "pywavelets" %}
+{% set version = "1.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6437af3ddf083118c26d8f97ab43b0724b956c9f958e9ea788659f6a2834ba93
+  sha256: ea027c70977122c5fc27b2510f0a0d9528f9c3df6ea3e4c577ca55fd00325a5b
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
+    - cython >=3.0.4,<3.1                    # [build_platform != target_platform]
+    - numpy >=2.0.0rc1,<2.3                  # [build_platform != target_platform]
+    - meson-python >=0.15.0
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
   host:
     - python
     - pip
-    - cython
-    - numpy
+    - cython >=3.0.4,<3.1
+    - numpy >=2.0.0rc1,<2.3
+    - meson-python >=0.15.0
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy >=1.22.4,<3
```

Comment:
Think [`numpy`'s `run_exports`]( https://github.com/conda-forge/numpy-feedstock/blob/96e4de9e6d71f1e8bb6b5525b9d82b946414f7a2/recipe/meta.yaml#L35-L36 ) will handle this for us

```suggestion
```